apps/website/docs/pipelines/release-container-app.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 1
+sidebar_label: Deploying App to Azure Container App
+---
+
+# Deploying Apps to Azure Container App
+
+This documentation covers the Container App Release reusable workflow defined in
+[release-azure-containerapp-v1.yaml](https://github.com/pagopa/dx/blob/main/.github/workflows/release-azure-containerapp-v1.yaml).
+
+## How It Works
+
+The workflow performs the following steps:
+
+1. Build and push the Docker image to the registry using the `docker-build-push`
+   action
+2. Determine the Container App’s revision mode: **Single** or **Multiple**
```

Comment:
suggestion: link to the related MS documentation

Reply:
Added